### PR TITLE
adding 3.0.0

### PR DESCRIPTION
**Pull request by voxxit**
3.0.0 was released today:
https://groups.google.com/forum/#!msg/redis-db/dO0bFyD_THQ/Uoo2GjIx6qgJ

### CODE REVIEW

**Comment by yosifkit** [comment]
Can we remove the unrelated changes in `2.6` and `2.8` as well as the unnecessary whitespace changes in the update script?  Just an amend and force push would work great :wink:.

**Comment by yosifkit** [comment]
Should probably also drop the `DEBIAN_FRONTEND` from `3.0` since the logs can be useful on the build server.

**Comment by voxxit** [comment]
@yosifkit how's that?

File: generate-stackbrew-library.sh [hunk 1]
```diff
@@ -17,14 +18,14 @@ echo '# maintainer: InfoSiftr <github@infosiftr.com> (@infosiftr)'
...
-	
+
 	versionAliases=()
 	while [ "$fullVersion" != "$version" -a "${fullVersion%[.-]*}" != "$fullVersion" ]; do
 		versionAliases+=( $fullVersion )
 		fullVersion="${fullVersion%[.-]*}"
 	done
 	versionAliases+=( $version ${aliases[$version]} )
-	
+
```
Comment (yosifkit) [inline generate-stackbrew-library.sh]: Can we leave these tabs in?  We like empty lines in our blocks with the same indentation (unless it is in go, then `gofmt -w -s` is right).

Comment (voxxit) [inline generate-stackbrew-library.sh]: Yes, sorry. That's atom.io's dirty work! :)

Comment (Starefossen) [inline generate-stackbrew-library.sh]: @yosifkit any particular reason for this rule? (just curious, as I have not encountered this before)

Comment (yosifkit) [inline generate-stackbrew-library.sh]: @Starefossen, We just like consistency in all of the `docker-library/` repos, so just a matter of preference ala [bikeshedding](http://bikeshed.io).  If there are inconsistencies, I blame @tianon for his laxed merging standards (it totally was never me :stuck_out_tongue_winking_eye:).

**Comment by yosifkit** [comment]
LGTM

**Comment by tianon** [comment]
LGTM

**Comment by stuartpb** [comment]
How soon until this hits the registry (available for pulling)?

**Comment by yosifkit** [comment]
Once [this PR](https://github.com/docker-library/official-images/pull/648) is merged and built; it is looking like it will be tomorrow morning.

**Pull request merged**